Misc/NEWS.d/next/Build/2025-03-01-18-20-07.gh-issue-130738.nDFSHR.rst

```diff
@@ -0,0 +1,2 @@
+Ensure that `Python.h` is included before `stdbool.h` unless `pyconfig.h` is included
```

Comment:
Ok actually this file should be deleted and the other should be kept (sorry, you committed at the same time I've reviewed). Just re-use that text though.